### PR TITLE
Changed "out_port_id" attribute setting in mapping file to store tensor names.

### PR DESCRIPTION
**Pull request by popovaan**
Description: 
Removed port id from 'fw_tensor_debug_info' attribute and changed "out_port_id" attribute setting in mapping file to store tensor names.
Fixed naming of Const nodes. 

Ticket: 51606

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names

Validation:
* [x]  Unit tests
* [x]  Framework operation tests - N/A
* [x]  Transformation tests - N/A
* [x]  Model Optimizer IR Reader check

Documentation:
* [x]  Supported frameworks operations list - N/A
* [x]  Guide on how to convert the **public** model - N/A
* [x]  User guide update - N/A

### CODE REVIEW

**Review by evolosen — APPROVED** [review]

**Review by achetver — APPROVED** [review]
Looks good to me

File: model-optimizer/mo/front/tf/extractor.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def create_tf_edge(src_node_id: str, dst_node_id: str, in_port: int):
         'in': in_port,
         'out': src_port,
         # debug anchor for a framework name, out port and tensor name
-        'fw_tensor_debug_info': [(src_node_id, src_port, tensor_name)],
+        'fw_tensor_debug_info': [(src_node_id, tensor_name)],
```
Comment (lazarevevgeny) [inline model-optimizer/mo/front/tf/extractor.py:41]: In OV 2021.2 the mapping file for TF is the following:

```xml
       <map>
           <framework name="InceptionV1/InceptionV1/Mixed_5c/Branch_0/Conv2d_0a_1x1/Conv2D" out_port_id="0"/>
           <IR name="InceptionV1/InceptionV1/Mixed_5c/Branch_0/Conv2d_0a_1x1/Conv2D" out_port_id="2" id="277"/>
       </map>
```

so the `out_port_id` is the port number and not the tensor name. So the PR description contradicts with the change.

Comment (lazarevevgeny) [reply model-optimizer/mo/front/tf/extractor.py:41]: May be we should change the behaviour and align it with all frameworks. @GlebKazantaev , what is the behavior in the serializer which you implemented?

Comment (GlebKazantaev) [reply model-optimizer/mo/front/tf/extractor.py:41]: Sure, we should produce the same mapping file using nGraph and MO legacy IR generation paths.
So, in nGraph transformation which generates mapping file I implemented next behavior:
```
For each node I iterate over output ports and generate next <map> blocks:
   <fw_name>, <fw_output_port_id>, <ir_name>, <ir_output_port_id>
1. node_name, node_name:output_port_id, node_name, ie_output_port_id
2. For each output port name: extracted_name, port_name, node_name, ie_output_port_id
```
I suggest to look at https://github.com/openvinotoolkit/openvino/pull/5347/files#diff-d5436d7c30193d976e9c350a0d04c208cd1ef95a0c243bf370c3931ae3e3585bR36
it is simple.
Also I validated this approach on e2e and DL Benchmark scopes.

Comment (GlebKazantaev) [reply model-optimizer/mo/front/tf/extractor.py:41]: offline discussion summary; https://github.com/openvinotoolkit/openvino/pull/5347#discussion_r621257289

Comment (popovaan) [reply model-optimizer/mo/front/tf/extractor.py:41]: Changed the logic of mapping generation for kaldi and mxnet according to offline discussion. 
Const naming was fixed.

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**